### PR TITLE
Introduce primitive type Bytes in ws language

### PR DESCRIPTION
**Pull request by wilmveel**

### CODE REVIEW

File: src/compiler/core/src/commonMain/kotlin/community/flock/wirespec/compiler/core/parse/EndpointParser.kt [hunk 1]
```diff
@@ -4,25 +4,7 @@ import arrow.core.Either
...
-import community.flock.wirespec.compiler.core.tokenize.types.QuestionMark
-import community.flock.wirespec.compiler.core.tokenize.types.RightCurly
-import community.flock.wirespec.compiler.core.tokenize.types.StatusCode
-import community.flock.wirespec.compiler.core.tokenize.types.WirespecType
-import community.flock.wirespec.compiler.core.tokenize.types.WsBoolean
-import community.flock.wirespec.compiler.core.tokenize.types.WsInteger
-import community.flock.wirespec.compiler.core.tokenize.types.WsNumber
-import community.flock.wirespec.compiler.core.tokenize.types.WsString
-import community.flock.wirespec.compiler.core.tokenize.types.WsUnit
+import community.flock.wirespec.compiler.core.tokenize.types.*
```
Comment (jerrevanveluw) [inline src/compiler/core/src/commonMain/kotlin/community/flock/wirespec/compiler/core/parse/EndpointParser.kt:7]: Don't use star imports. In IntelliJ you can configure for Kotlin and Java either don't use star imports or set the limit to 9999 or something high.

File: src/compiler/core/src/commonMain/kotlin/community/flock/wirespec/compiler/core/parse/TypeParser.kt [hunk 2]
```diff
@@ -4,26 +4,7 @@ import arrow.core.Either
...
-import community.flock.wirespec.compiler.core.tokenize.types.RightCurly
-import community.flock.wirespec.compiler.core.tokenize.types.TypeDefinitionStart
-import community.flock.wirespec.compiler.core.tokenize.types.WirespecDefinition
-import community.flock.wirespec.compiler.core.tokenize.types.WirespecType
-import community.flock.wirespec.compiler.core.tokenize.types.WsBoolean
-import community.flock.wirespec.compiler.core.tokenize.types.WsInteger
-import community.flock.wirespec.compiler.core.tokenize.types.WsNumber
-import community.flock.wirespec.compiler.core.tokenize.types.WsString
-import community.flock.wirespec.compiler.core.tokenize.types.WsUnit
+import community.flock.wirespec.compiler.core.tokenize.types.*
```
Comment (jerrevanveluw) [inline src/compiler/core/src/commonMain/kotlin/community/flock/wirespec/compiler/core/parse/TypeParser.kt:7]: No * import

File: src/compiler/lib/src/jsMain/kotlin/community/flock/wirespec/compiler/lib/Ast.kt [hunk 3]
```diff
@@ -233,6 +233,7 @@ private fun Reference.Primitive.Type.produce() = when (this) {
     is Reference.Primitive.Type.Integer -> WsPrimitiveType.Integer
     is Reference.Primitive.Type.Number -> WsPrimitiveType.Number
     is Reference.Primitive.Type.Boolean -> WsPrimitiveType.Boolean
+    is Reference.Primitive.Type.Bytes -> WsPrimitiveType.String
```
Comment (jerrevanveluw) [inline src/compiler/lib/src/jsMain/kotlin/community/flock/wirespec/compiler/lib/Ast.kt:236]: Is `WsPrimitiveType.String` correct?

File: examples/maven-spring-compile/src/test/java/community/flock/wirespec/example/maven/custom/app/user/UserTest.java [hunk 4]
```diff
@@ -2,12 +2,10 @@
...
 import org.junit.jupiter.api.Test;
 
+import java.util.Random;
 import java.util.function.Consumer;
 
-import static community.flock.wirespec.example.maven.custom.app.user.UserContext.Service.deleteUserByName;
-import static community.flock.wirespec.example.maven.custom.app.user.UserContext.Service.getAllUsers;
-import static community.flock.wirespec.example.maven.custom.app.user.UserContext.Service.getUserByName;
-import static community.flock.wirespec.example.maven.custom.app.user.UserContext.Service.saveUser;
+import static community.flock.wirespec.example.maven.custom.app.user.UserContext.Service.*;
```
Comment (jerrevanveluw) [inline examples/maven-spring-compile/src/test/java/community/flock/wirespec/example/maven/custom/app/user/UserTest.java:8]: no * import

File: src/compiler/lib/src/jsMain/kotlin/community/flock/wirespec/compiler/lib/Ast.kt (hunk 3 above)
Comment (wilmveel) [reply src/compiler/lib/src/jsMain/kotlin/community/flock/wirespec/compiler/lib/Ast.kt]: Incorrect

**Review by jerrevanveluw — APPROVED** [review]

**Pull request merged**